### PR TITLE
Update Vue 2 recipe

### PR DESCRIPTION
**Pull request by amypellegrini**
Added context on XState usage depending on the version of Vue.

### CODE REVIEW

File: docs/recipes/vue.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Usage with Vue
 
+Usage of Xstate in a Vue application may vary depeding on which version of Vue your application is running. This page focuses on Vue 2 only. To see how to use Xstate with Vue 3 check the documentation for the Xstate [`@xstate/vue` package](../packages/xstate-vue).
```
Comment (Andarist) [inline docs/recipes/vue.md:3]:
```suggestion
Usage of XState in a Vue application may vary depending on which version of Vue your application is running. This page focuses on Vue 2 only. To see how to use XState with Vue 3 check the documentation for the XState [`@xstate/vue` package](../packages/xstate-vue).
```

File: docs/recipes/vue.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+
+There are two ways you can use Xstate with Vue 2:
```
Comment (Andarist) [inline docs/recipes/vue.md:5]:
```suggestion
There are two ways you can use XState with Vue 2:
```

File: docs/recipes/vue.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+
+1. Using the `useMachine` hook provided by the [`xstate-vue2` package](https://github.com/ChrisShank/xstate-vue2) (3rd-party) plugin;
+2. Using Xstate `interpret` utility to create a service and inject it into your app.
```
Comment (Andarist) [inline docs/recipes/vue.md:8]:
```suggestion
2. Using XState `interpret` utility to create a service and inject it into your app.
```

**Comment by Andarist** [comment]
I've noticed that it's still in the draft. Is there any other information that you plan to add to this PR?

**Comment by amypellegrini** [comment]
Hi @Andarist yes, I intend to work on the whole page so I think it makes sense to wait until the whole editing is done before merging, instead of having multiple PR's. Unless you think it's better otherwise, just let me know.

File: docs/recipes/vue.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Usage with Vue
 
+Usage of XState in a Vue application may vary depending on which version of Vue your application is running. This page focuses on Vue 2 only. To see how to use XState with Vue 3 check the documentation for the XState [`@xstate/vue` package](../packages/xstate-vue).
+
+There are two ways you can use XState with Vue 2:
+
+1. Using the `useMachine` hook provided by the [`xstate-vue2` package](https://github.com/ChrisShank/xstate-vue2) (3rd-party) plugin;
```
Comment (ChrisShank) [inline docs/recipes/vue.md:7]: Might be worth specifying this is `^2.7`?

**Comment by amypellegrini** [comment]
I made the distinction between using the external plugin or using `interpret` more explicit. I'm not sure if it is possible to make use of the `setup` feature from Vue for the `interpret` example:

https://vuejs.org/api/sfc-script-setup.html

**Comment by amypellegrini** [comment]
I'm not sure if I should add a change set for this, but my best guess is that docs are not versioned since they are not inside the packages folder. Please let me know if I should add a change set, I try to learn more about how the docs are built so I can make PR's easier to merge.

**Review by davidkpiano — APPROVED** [review]

**Pull request merged**